extensions/wikia/ArticleVideo/scripts/article-featured-video.js

```diff
@@ -37,9 +52,13 @@ require([
 				autoplay = abTest.inGroup('FEATURED_VIDEO_AUTOPLAY', 'AUTOPLAY') && window.OO.allowAutoPlay,
 				vastUrl;
 
+			if (playerTracker) {
+				playerTracker.track(playerTrackerParams, 'init');
+			}
+
 			if (vastUrlBuilder && adContext && adContext.getContext().opts.showAds) {
 				vastUrl = vastUrlBuilder.build(640/480, {
-					pos: 'FEATURED_VIDEO',
+					pos: playerTrackerParams.slotName,
```

Comment:
I don't feel great when we're using `playerTrackerParams` for ads targeting. Maybe we should have a separate variable for ad slot name that we can use in both places?